Airports/EGPE/Sector.txt

```diff
@@ -8,19 +8,29 @@ DISPLAY:Moray:Moray:Only PEAPP
 DISPLAY:Hebrides:Hebrides:Only PEAPP
 DISPLAY:Montrose:Montrose:Only PEAPP
 
+CIRCLE_SECTORLINE:PEAPP:EGPE:10
+
 CIRCLE_SECTORLINE:PETWR:EGPE:2.5
 DISPLAY:PEAPP:PEAPP:PETWR
 DISPLAY:PETWR:PEAPP:PETWR
 
-SECTORLINE:PEAPP
+SECTORLINE:PEAPP-N560
+DISPLAY:PEAPP:Moray:PEAPP
+COORD:N056.52.56.512:W004.21.18.447
+COORD:N057.30.00.477:W004.12.32.557
+COORD:N057.45.12.293:W004.23.48.009
+COORD:N057.54.46.705:W004.32.05.128
+COORD:N058.08.07.327:W004.03.45.530  
+COORD:N058.00.42.237:W003.51.07.701 
+COORD:N057.52.11.320:W004.09.07.003
+COORD:N057.33.13.757:W003.52.58.280
```

Comment:
```suggestion
COORD:N057.33.17.000:W003.52.55.000
```

Reply:
Because this...

![image](https://user-images.githubusercontent.com/12017284/91491733-63868280-e8ac-11ea-8009-3f51f96911f6.png)

Is better than...

![image](https://user-images.githubusercontent.com/12017284/91491706-59648400-e8ac-11ea-8396-2628e4167d19.png)

😉